From 40dd38388c3d1c87efe254cee533fc5db5ffc4ed Mon Sep 17 00:00:00 2001
From: Axel Gembe <ago@bastart.eu.org>
Date: Wed, 14 May 2008 12:43:34 +0200
Subject: [PATCH] bcm963xx: add missing trailing zero to load address

The load address for BCM963xx is 0x80010000, not 0xf8001000 as in the current
sources. I think this is just a typo, so this patch fixes it (tested on 96345).

Signed-off-by: Axel Gembe <ago@bastart.eu.org>
---
 arch/mips/Makefile |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

Index: linux-2.6.25.4/arch/mips/Makefile
===================================================================
--- linux-2.6.25.4.orig/arch/mips/Makefile
+++ linux-2.6.25.4/arch/mips/Makefile
@@ -562,7 +562,7 @@ load-$(CONFIG_BCM47XX)		:= 0xffffffff800
 
 core-$(CONFIG_BCM963XX)		+= arch/mips/bcm963xx/
 cflags-$(CONFIG_BCM963XX)	+= -Iinclude/asm-mips/mach-bcm963xx
-load-$(CONFIG_BCM963XX)		:= 0xffffffff8001000
+load-$(CONFIG_BCM963XX)		:= 0xffffffff80010000
 
 #
 # SNI RM
